packages/api/src/api/expoJudgingVote/post.ts

```diff
@@ -0,0 +1,46 @@
+import { Schema } from '@hangar/shared';
+import { Request, Response } from 'express';
+import { ExpoJudgingSession, ExpoJudgingVote, Judge } from '@hangar/database';
+import { validatePayload } from '../../utils/validatePayload';
+import { logger } from '../../utils/logger';
+
+export const post = async (req: Request, res: Response) => {
+  const { entityManager, judge } = req;
+  
+  const { errorHandled, data } = validatePayload({
+    req,
+    res,
+    schema: Schema.expoJudgingVote.post,
+  });
+  if (errorHandled) return;
+
+  const { currentProjectChosen, expoJudgingSessionId } = data
+
```

Comment:
One thing I forgot to mention that's critical... _**does the judge have permissions to cast a vote in that judging session?**_

You should be able to do something along the lines of 
``` typescript
judge.expoJudgingSessions.getIdentifiers().includes(expoJudgingSessionId)
````